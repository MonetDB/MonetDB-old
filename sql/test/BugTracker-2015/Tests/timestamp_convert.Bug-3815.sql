SELECT (str_to_timestamp('2015-10-01', '%Y-%m-%d') - str_to_timestamp('2015-09-30', '%Y-%m-%d'))/86400;

SELECT (str_to_timestamp('2015-10-01', '%Y-%m-%d') - str_to_timestamp('2015-09-30', '%Y-%m-%d'))/86400;

SELECT (str_to_timestamp('2015-10-01', '%Y-%m-%d') - str_to_timestamp('2015-09-30', '%Y-%m-%d'))/86400.0;

SELECT date '2015-03-01' - (date '2015-02-01' - date '2015-01-01');
