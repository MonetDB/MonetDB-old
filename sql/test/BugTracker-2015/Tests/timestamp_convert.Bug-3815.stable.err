stderr of test 'timestamp_convert.Bug-3815` in directory 'sql/test/BugTracker-2015` itself:


# 19:04:37 >  
# 19:04:37 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=33045" "--set" "mapi_usock=/var/tmp/mtest-17747/.s.monetdb.33045" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/scratch/niels/build/rc/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2015" "--set" "mal_listing=0" "--set" "embedded_r=yes"
# 19:04:37 >  


# 19:04:38 >  
# 19:04:38 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-17747" "--port=33045"
# 19:04:38 >  


# 19:04:38 >  
# 19:04:38 >  "Done."
# 19:04:38 >  

