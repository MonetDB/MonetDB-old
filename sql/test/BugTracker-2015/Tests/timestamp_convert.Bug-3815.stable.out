stdout of test 'timestamp_convert.Bug-3815` in directory 'sql/test/BugTracker-2015` itself:


# 19:04:37 >  
# 19:04:37 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=33045" "--set" "mapi_usock=/var/tmp/mtest-17747/.s.monetdb.33045" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/scratch/niels/build/rc/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2015" "--set" "mal_listing=0" "--set" "embedded_r=yes"
# 19:04:37 >  

#SELECT (str_to_timestamp('2015-10-01', '%Y-%m-%d') - str_to_timestamp('2015-09-30', '%Y-%m-%d'))/86400;
% .%2 # table_name
% %2 # name
% sec_interval # type
% 5 # length
[ 1.000	]
#SELECT (str_to_timestamp('2015-10-01', '%Y-%m-%d') - str_to_timestamp('2015-09-30', '%Y-%m-%d'))/86400;
% .%2 # table_name
% %2 # name
% sec_interval # type
% 5 # length
[ 1.000	]
#SELECT (str_to_timestamp('2015-10-01', '%Y-%m-%d') - str_to_timestamp('2015-09-30', '%Y-%m-%d'))/86400.0;
% .%2 # table_name
% %2 # name
% sec_interval # type
% 5 # length
[ 1.000	]
#SELECT date '2015-03-01' - (date '2015-02-01' - date '2015-01-01');
% .%2 # table_name
% %2 # name
% date # type
% 10 # length
[ 2015-01-29	]

# 19:04:38 >  
# 19:04:38 >  "Done."
# 19:04:38 >  

